### ISSUE TITLE
Basic data model for a user

### ISSUE THREAD

**Issue by masoug** [scalableinternetservices/Luber#11]
For now we have username, password, and email in the user model. This issue is the discussion for what other details/fields should be part of the user model.

**Comment by masoug** [comment]
@Heronalps Can you post a picture on this thread of the user model we made last Thursday? Also can you summarize your thoughts from the `user_model` branch?

Once we settle on a schema, I will go ahead and fork/migrate/submit a pull request for the changes.

**Comment by justinpearson** [comment]
Good work Michael: user_model branch
